includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -527,8 +531,9 @@ private function get_courses( $args ) {
 	/**
 	 * Return array of lessons
 	 *
-	 * @since  1.7.0
+	 * @param object $args Arguments for query.
```

Comment:
I think this param type should be an array?

Reply:
You're right, updated in https://github.com/Automattic/sensei/pull/4883/commits/924df553acadf51176832236234004ca471760bf